scripts/build-rir.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -xe
+
+SRC_DIR=${SRC_DIR:-$PWD}
+BUILD_DIR=${BUILD_DIR:-$PWD/rkt-build}
+RKT_BUILDER_ACI=${RKT_BUILDER_ACI:-coreos.com/rkt/builder:v0.10.0}
+
+mkdir -p $BUILD_DIR
+
+rkt run \
+    --volume src-dir,kind=host,source=$SRC_DIR \
+    --volume build-dir,kind=host,source=$BUILD_DIR \
+    --interactive \
+    $RKT_BUILDER_ACI \
+    --exec /bin/bash -- -c 'cd /opt/rkt && ./autogen.sh && ./configure && make'
```

Comment:
Is this `--exec` line really necessary? It is already specified like this in manifest.

Also, missing newline at the end of the file.